content/200-orm/100-prisma-schema/10-overview/index.mdx

```diff
@@ -167,12 +175,12 @@ The Prisma CLI looks for the Prisma schema file in the following locations, in t
     - `./prisma/schema.prisma`
     - `./schema.prisma`
```

Comment:
Probably missing the new default folder location(s) here. Is it only `./prisma/schema/*` or also `./schema/*` @SevInf?